src/trade.js

```diff
@@ -284,6 +295,7 @@ class Trade {
       // Only consider transactions that are complete or that we're still
       // expecting payment for:
       return [
+        'awaiting_reference_number',
```

Comment:
The state names in `filteredTrades` `bitcoin-exchange-client` are inspired by Coinify, but they are universal. The states from other exchanges are mapped to these. E.g. the SFOX state "ready" is mapped to "processing". It might make sense to implement `filteredTrades` in each subclass instead, but so far I find states to be similar enough between exchanges.